lib/src/modules/conditional.dart

```diff
@@ -1274,18 +1288,21 @@ class FlipFlop extends Module with CustomSystemVerilog {
 
   /// Performs setup for custom functional behavior.
   void _setup() {
-    Sequential(_clk, [q < _d]);
+    Sequential(_clk, [
```

Comment:
The design here is correct when `en` is provided, but we don't need to generate hardware that conditionally checks if `en` is high when it was _not_ provided.  Here, we want to conditionally generate the `If` around the conditional assignment only if `en` was provided.

This will make the ROHD simulation (slightly) more efficient by not needing to redundantly check the condition.  It will also make the generated outputs like SystemVerilog cleaner (even though the synthesis tools will probably optimize the if(1) away).